.nvmrc

```diff
@@ -0,0 +1 @@
+v16
```

Comment:
The [eslint.yml](https://github.com/10up/retro-winamp-block/blob/59ef1614ecae2119c5b89fbd0d6183d277ed7e8f/.github/workflows/eslint.yml#L21) workflow is using hardcoded Node version, should we refer to this `.nvmrc` file instead?

Reply:
@cadic this is done!